### PR TITLE
FIX travis failure: Use new cask address for homebrew

### PR DESCRIPTION
**Pull request by jni**
## Description

See #4212 for details. This may or may not be considered a fix for that issue.

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by jni** [comment]
This doesn't change any code so if the Travis builds pass, please just merge!

**Comment by lagru** [comment]
@jni There seems to be another issue :(
```
...
+++brew tap homebrew/cask-cask
==> Tapping homebrew/cask-cask
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask-cask'...
Username for 'https://github.com': 

No output has been received in the last 10m0s, this potentially
indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: 
https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

The build has been terminated
```

**Comment by jni** [comment]
Well, I'm out of ideas...

https://travis-ci.org/scikit-image/scikit-image/jobs/594501755#L2297-L2300

**Comment by jni** [comment]
LOL Sorry, missed github update. Yup... =(

**Comment by lagru** [comment]
> I know nothing about osx.

Likewise. But is it possible that brew tries to clone into a GitHub repository it doesn't have access to? That would explain the login prompt. 

It's strange that I can't find `cask-cask` in [homebrews formulae](https://formulae.brew.sh/formula/). But `cask` is available. Maybe we could try `brew tap homebrew/cask` instead?

Maybe someone with access to MacOS can use `brew tap-info` to get more information?

File: tools/travis/osx_install.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -ex
 
 brew update
 brew install ccache
-brew tap caskroom/cask
+brew tap homebrew/cask-cask
```
Comment (soupault) [inline tools/travis/osx_install.sh]: Try `brew tap Homebrew/homebrew-cask`

Comment (rfezzani) [inline tools/travis/osx_install.sh]:
```suggestion

```
See my [comment](https://github.com/scikit-image/scikit-image/pull/4219#discussion_r332630937) ;-)

**Comment by NelleV** [comment]
@stefanv (related to the slack message I left you). Any chance you could figure out the correct cask homebrew address using the mac from BIDS?

**Comment by soupault** [comment]
@NelleV see https://github.com/scikit-image/scikit-image/pull/4213#discussion_r332516471. Worked for me.

**Comment by NelleV** [comment]
@soupault Testing on my PR!

**Comment by stefanv** [comment]
```
==> Tapping homebrew/cask-cask

Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask-cask'...

Username for 'https://github.com': 
```

**Comment by NelleV** [comment]
@stefanv we found the solution. It's in #4219

**Comment by NelleV** [comment]
I'm closing this PR as the fix has been merged into master. Thanks @jni for this attempt!

**Pull request closed without merging**